hosting/learnings-from-2025.md

```diff
@@ -0,0 +1,80 @@
+# Learnings from the 2025 Hackathon
+
+All in all, this was surprisingly successful. There were some technical issues though.
+
+## Overall organization
+
+* Andrew, Pier-Luigi, Sara and Yuting did a great job at holding things together
+* Bjorn did a great job at shaking up structures and pushing things forward
+* Tobi, Lukas, Mark M, and others made technical magic possibble.
+
+## Schedule
+
+## Inter-node communication
+
+### Mattermost
+
+* Works
+* Invite more people earlier in the process
+* China uses WeChat for most communication (in Chinese, but with auto-translation)
+
+### Zoom calls during the preparation phase
+
+* Double Telcos are tiring but necessary.
+* Did anybody read the nodes / watch the recordings?
```

Comment:
My educated guess is: no.

And I think it would be wise to actually reassess the value of those. I think it is nice to have both live streams and video calls between nodes, but their goal should be absolutely clear.

If the goal of a live stream is to merely great the other participants and create a feeling of collaborative work **in the moment** we don't need a recording.

If the goal of a video call is to get everyone on the same page and divide the tasks **to be tackled next** we don't (necessarily) need notes.